projects/opendr_ws/README.md

```diff
@@ -73,12 +73,13 @@ Currently, apart from tools, opendr_ws contains the following ROS nodes (categor
 5. [2D Object Detection](src/opendr_perception/README.md#2d-object-detection-ros-nodes)
 6. [2D Single Object Tracking](src/opendr_perception/README.md#2d-single-object-tracking-ros-node)
 7. [2D Object Tracking](src/opendr_perception/README.md#2d-object-tracking-ros-nodes)
-8. [Panoptic Segmentation](src/opendr_perception/README.md#panoptic-segmentation-ros-node)
-9. [Semantic Segmentation](src/opendr_perception/README.md#semantic-segmentation-ros-node)
-10. [Image-based Facial Emotion Estimation](src/opendr_perception/README.md#image-based-facial-emotion-estimation-ros-node)
-11. [Landmark-based Facial Expression Recognition](src/opendr_perception/README.md#landmark-based-facial-expression-recognition-ros-node)
-12. [Skeleton-based Human Action Recognition](src/opendr_perception/README.md#skeleton-based-human-action-recognition-ros-node)
-13. [Video Human Activity Recognition](src/opendr_perception/README.md#video-human-activity-recognition-ros-node)
+8. [Vision Based Panoptic Segmentation](src/opendr_perception/README.md#vision-based-panoptic-segmentation-ros-node)
+9. [LiDAR Based Panoptic Segmentation](src/opendr_perception/README.md#lidar-based-panoptic-segmentation-ros-node)
```

Comment:
Please, revert the changes in this list and move the new node entry to point cloud input node list below.